loops/lang/en-EN/game.xml

```diff
@@ -1769,7 +1769,7 @@
         <spatiomancy>
             <label>Spatiomancy</label>
             <tooltip><![CDATA[
-            At a base level, this is the art of manipulating matter. At a high level, well, this is that but more powerful. Being able to manipulate space itself will allow you to create more items for certain limited actions. Mana cost is reduced by Wizard College Grade.
+            At a base level, this is the art of manipulating matter. At a high level, well, this is that but more powerful. Being able to manipulate space itself will allow you to create more items for certain limited actions.<br>Mana cost is reduced by Wizard College Grade.
```

Comment:
Why does manipulating space imply that you can create more items for limited actions? I like the mechanic but I feel like the motivation and story aren't quite there.

This skill feels more like you are 'blending' multiple worlds together. This town only has so many small problems that need dealing with (Short Quests) - but an almost identical, parallel town will have similar but different problems.

This skill could allow you to 'shift' between this and another parallel world, allowing you to do more of the same thing in better ways. Doesn't quite work for Buy Pickaxe, but maybe there is a way to tie it in.

It's hard to communicate big ideas in small spaces, and probably counterproductive (better to suggest things and let conclusions be drawn), but I've had a go at something that might work (and is not explicit at all):

```suggestion
            Learn how to control space and matter. Turns out making big changes is <i>hard</i>, but even small changes might allow you to get more done each loop.
            <br>Mana cost is reduced by Wizard College Grade.
```